docs/ManualDeploymentGuideScalarDLAuditorOnAWS.md

```diff
@@ -59,28 +59,18 @@ so you need to add peering for internal communication between the Auditor, Ledge
 
 ### Steps
 
-* Create 3 peering connections between the Ledger, Auditor and Client VPCs based on the [AWS official guide](https://docs.aws.amazon.com/vpc/latest/peering/create-vpc-peering-connection.html).
+* Create three peering connections between the Ledger, Auditor and Client VPCs based on the [AWS official guide](https://docs.aws.amazon.com/vpc/latest/peering/create-vpc-peering-connection.html).
     * Peering between Ledger and Auditor.
     * Peering between Ledger and Client.
     * Peering between Auditor and Client.
 * Update route tables for VPC peering connection based on the [AWS official guide](https://docs.aws.amazon.com/vpc/latest/peering/vpc-peering-routing.html).
-* Add new Inbound and Outbound rule to restrict unwanted access to Ledger and Auditor.
-    * Add new Inbound rules to the Ledger Network ACLs to allow the Ledger Envoy LoadBalancer (e.g., 50051 and 50052 by default) access from the Auditor and Client.
+* Add new Inbound and Outbound rules to Ledger Network ACLs and Auditor Network ACLs to restrict unwanted access to the Ledger and Auditor.
```

Comment:
English grammar thing.
If we use `Ledger` as a proper noun, don't do like `the Ledger`.
On the other hand, if you specify something of Ledger (e.g., Ledger Network ACLs), then you should add an article like `the Ledger Network ACLs`.